### DISCUSSION TITLE
benchmark test for DoD STIG

### DISCUSSION THREAD

**Discussion by cpt-redbeard** [aquasecurity/kube-bench#833] [Development]
DoD has released a publicly available Kubernetes benchmarking test in XML format. This could be useful as an additional POV for security?

The document can be found here: [https://public.cyber.mil/devsecops/](https://public.cyber.mil/devsecops/) under the “Kubernetes Draft STIG – Ver 1, Rel 0.1”

**Comment by yoavrotems** [comment]
Hey @cpt-redbeard it could be useful depends on your needs, I did a comparison a few years ago with STIG in other case we had a saw there was around 90% overlapping. 
So currently we decided it won't get in our assignment pool...
But we encourage our community to step up an contribute if you see a need for and want to PR it you are welcome! and I will be for any question 😸
